docs/generate-time-series/go.rst

```diff
@@ -380,27 +408,30 @@ will open up a map view showing the current position of the ISS:
 
 .. image:: ../_assets/img/generate-time-series/map.png
 
-You can find the full version of the script `here`_. 
+.. TIP::
```

Comment:
@infoverload added this to match the same addition to other tutorials I made in this changeset